Sources/Orbit/Support/Accessibility/AccessibilityId.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+
+/// Orbit accessibility identifier for use in wrapper components.
+public enum AccessibilityId: String {
+    case alertButtonPrimary             = "orbit.alert.button.primary"
+    case alertButtonSecondary           = "orbit.alert.button.secondary"
+    case alertTitle                     = "orbit.alert.title"
+    case alertDescription               = "orbit.alert.description"
+    case cardTitle                      = "orbit.card.title"
+    case cardDescription                = "orbit.card.description"
+    case cardActionButtonLink           = "orbit.card.action.buttonLink"
+    case checkboxTitle                  = "orbit.checkbox.title"
+    case checkboxDescription            = "orbit.checkbox.description"
+    case choiceTileTitle                = "orbit.choicetile.title"
+    case choiceTileDescription          = "orbit.choicetile.description"
+    case choiceTileBadge                = "orbit.choicetile.badge"
+    case dialogTitle                    = "orbit.dialog.title"
+    case dialogDescription              = "orbit.dialog.description"
+    case dialogButtonPrimary            = "orbit.dialog.button.primary"
+    case dialogButtonSecondary          = "orbit.dialog.button.secondary"
+    case dialogButtonTertiary           = "orbit.dialog.button.tertiary"
+    case emptyStateTitle                = "orbit.emptystate.title"
+    case emptyStateDescription          = "orbit.emptystate.description"
+    case emptyStateButton               = "orbit.emptystate.button"
+    case fieldMessage                   = "orbit.field.message"
+    case inputPrefix                    = "orbit.input.prefix"
+    case inputSuffix                    = "orbit.input.suffix"
+    case inputValue                     = "orbit.input.value"
+    case inputFieldClear                = "orbit.inputfield.clear"
+    case inputFieldPasswordToggle       = "orbit.inputfield.password.toggle"
+    case keyValueKey                    = "orbit.keyvalue.key"
+    case keyValueValue                  = "orbit.keyvalue.value"
+    case listChoiceTitle                = "orbit.listchoice.title"
+    case listChoiceDescription          = "orbit.listchoice.description"
+    case passwordStrengthIndicator      = "orbit.passwordstrengthindicator"
+    case radioTitle                     = "orbit.radio.title"
+    case radioDescription               = "orbit.radio.description"
+    case tileTitle                      = "orbit.tile.title"
+    case tileDescription                = "orbit.tile.description"
+    case tileDisclosureButtonLink       = "orbit.tile.disclosure.buttonlink"
+    case tileDisclosureIcon             = "orbit.tile.disclosure.icon"
```

Comment:
The raw values are public if done this way. Don't we want to make them internal by moving them into a computed `var`?

Reply:
This was intentional, so that caller in UI test code (or PageObject creator) can directly use these to define selectors and assertions.